client/me/help/help-contact-closed/index.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormSectionHeading from 'components/forms/form-section-heading';
+import HelpContactClosedDetail from './detail';
+
+export default localize( ( props ) => {
+	const {	translate } = props;
+
+	return (
+		<div className="help-contact-closed">
+			<FormSectionHeading>{ translate( 'Limited Support This Week' ) }</FormSectionHeading>
+			<p>
+				{ translate(
+					'Private email and live chat support will be closed from %(closed_start_date)s through %(closed_end_date)s, ' +
+					'included. We will reopen private email support on %(email_open_date)s, and live chat support will be back as ' +
+					'usual on %(livechat_open_date)s.', {
+						args: {
+							closed_start_date: 'Wednesday, September 14th, 2016',
```

Comment:
Ideally, we'd localize these dates. Maybe we can use a timestamp and do a proper conversion rather than just a string substitution? For example, even within English, I think in some cases this'd be displayed as "Wednesday, 14th September, 2016," and I'll be other languages vary even more widely (or at any rate, have different month and day names).

I'm ok with shipping the PR without this change, but we should follow up shortly to address.